policy-descriptor/examples/policy-example.yml

```diff
@@ -0,0 +1,23 @@
+# do NOT delete: this is used by tng-sdk-descriptorgen
+# default YAML description of an ubuntu vm
```

Comment:
you can delete these two comments as they only apply to the default VNFD and NSD at the moment